telebot/types.py

```diff
@@ -776,6 +833,9 @@ class Message(JsonDeserializable):
     :param forward_date: Optional. For forwarded messages, date the original message was sent in Unix time
     :type forward_date: :obj:`int`
 
+    :forward_origin: Optional. For forwarded messages, information about the original message; 
```

Comment:
Comment for me: set deprecation for forward_xxx fields (in the next update: this one is too large).

_Added the class [MessageOrigin](https://core.telegram.org/bots/api#messageorigin) and replaced the fields forward_from, forward_from_chat, forward_from_message_id, forward_signature, forward_sender_name, and forward_date with the field forward_origin of type [MessageOrigin](https://core.telegram.org/bots/api#messageorigin) in the class [Message](https://core.telegram.org/bots/api#message)._